scripts/chart/test

```diff
@@ -3,8 +3,7 @@
 echo "-- chart/test --"
 
 # Check for helm
-hash helm >/dev/null 2>&1
-if [[ $? > 0 ]]; then
+if [ -z "$(type -p helm)" ]; then
```

Comment:
Any reason why we can't use helm_v3 here? Plugins should still be supported in Helm 3

Reply:
tried this out locally and I see that there seems to be an issue with the way rancher/helm-unittest parses the ARCH when used as a plugin for Helm 3... resolving this comment.